arch/x86/core/crt0.S

```diff
@@ -415,6 +415,12 @@ __csSet:
 1:
 #endif
 
+#if defined(CONFIG_PIC_DISABLE)		/* disable i8259s if present */
+	movb $0xff, %al
+	outb %al, $0x21
```

Comment:
just add come comment about what is being done here and where these numbers came from, for example:
```

    /* Disable PIC command */
   	movb $0xff, %al
    /* Port 2, master base address (0x20) */
    outb %al, $0x21
    /* Port 2, slave base address (0xa0) */
	outb %al, $0xA1
```

Reply:
@nashif I've added some explanatory comments.